Xwt.XamMac/Xwt.Mac/TextLayoutBackendHandler.cs

```diff
@@ -225,8 +226,9 @@ void AddAttributeInternal (TextAttribute attribute)
 					var xa = (FontSizeTextAttribute)attribute;
 					NSRange er;
 					var ft = TextStorage.GetAttribute (NSStringAttributeKey.Font, attribute.StartIndex, out er, r) as NSFont;
```

Comment:
This is a bit confusing. If we already have a font attribute there, why are we not modifying the existing one, but adding a new one?

Reply:
No idea. I was only interested in fixing the null ref. Re-using the same font would presumably have the same problem?